portfolio/src/main/webapp/index.html

```diff
@@ -7,13 +7,47 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="container"> 
+               
```

Comment:
Indentation should be by 2 spaces at a time. See: https://google.github.io/styleguide/htmlcssguide.html#Indentation.